backend/apps/events/views.py

```
@@ -91,47 +250,97 @@ def post(self, request):
+                    location=location if event_type == Event.Type.EXTERNAL else '',
+                    space=space,
+                    reservation=reservation,
+                    image_url=body.get('image_url'),
+                    tags=body.get('tags'),
+                    max_participants=normalized_max_participants,
+                    residence=request.residence,
+                    host=request.user
+                )
+                EventParticipation.objects.create(event=event, user=request.user)
             return JsonResponse({'id': event.id, 'detail': 'Event created successfully'}, status=201)
         except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)